### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Afsheen96**
i added required changes 

- Mobile first version
- add animation and transition 
- add desktop media query
- add README file
- add live demo link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+      If you have an application you are interested in developing, a feature
+      that you need built or a project that needs coding. I'd love to help
+      with it
+    </p>
+    <form action="https://formspree.io/f/xpzebdqb" method="post" class="contact-form">
+      <input type="text" maxlength="30" placeholder="Enter your name here" required class="form-input">
+      <input type="email" pattern="[^ @]*@[^ @]*" placeholder="Enter your Email here" required class="form-input">
+      <textarea cols="30" rows="6" placeholder="Write your message here" maxlength="500" required
+        class="form-input"></textarea>
+      <button class="form-button" type="submit">
```
Comment (bellom) [inline index.html:233]: I was unable to submit the form, kindly fix this and that should be all.

**Review by bellom — CHANGES_REQUESTED** [review]
@Afsheen96,

### This is a manual review

# Request Changes :+1:

check the comment below.

Cheers and Happy coding!:clap::clap::clap:

File: index.html (hunk 1 above)
Comment (Afsheen96) [reply index.html:233]: form submit is not the requirement of this project as this activity does not need javascript functionality for this milestone of project @bellom

**Comment by Afsheen96** [comment]
This was not the requirement for this milestone ... as  As it is clearly mentioned that we don’t need to implement any js functionality. So how can I enable form submitting   On Fri, 23 Jun 2023, 5:07 pm Oluwaseun Bello, ***@***.***> ***@***.*** requested changes on this pull ***@***.***,This is a manual reviewRequest Changes 👍check the comment below.Cheers and Happy coding!👏👏👏In index.html:> +      </ul>+    </div>+  </section>+  <section class="form-container">+    <h1 class="form-heading">Contact me</h1>+    <p class="form-paragraph">+      If you have an application you are interested in developing, a feature+      that you need built or a project that needs coding. I'd love to help+      with it+    </p>+    <form action="https://formspree.io/f/xpzebdqb" method="post" class="contact-form">+      <input type="text" maxlength="30" placeholder="Enter your name here" required class="form-input">+      <input type="email" pattern="[^ @]*@[^ @]*" placeholder="Enter your Email here" required class="form-input">+      <textarea cols="30" rows="6" placeholder="Write your message here" maxlength="500" required+        class="form-input"></textarea>+      <button class="form-button" type="submit">I was unable to submit the form, kindly fix this and that should be all.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

File: index.html (hunk 1 above)
Comment (bellom) [reply index.html:233]: > form submit is not the requirement of this project as this activity does not need javascript functionality for this milestone of project @bellom

I think you later figure it out, good job 👍.

**Comment by bellom** [comment]
Hi @Afsheen96 

### This is a manual review

# Approved :+1:

Cheers and Happy coding!:clap::clap::clap:

File: index.html (hunk 1 above)
Comment (Afsheen96) [reply index.html:233]: thank you yeah i applied the name attribute for the required issue

**Comment by bellom** [comment]
@Afsheen96 - Please go ahead and merge!

**Pull request merged**